osu.Game/Graphics/Containers/UserDimContainer.cs

```diff
@@ -53,11 +53,11 @@ public abstract partial class UserDimContainer : Container
 
         protected Bindable<bool> LightenDuringBreaks { get; private set; } = null!;
 
-        protected Bindable<bool> ShowStoryboard { get; private set; } = null!;
+        protected Bindable<bool> ShowStoryboardConfig { get; private set; } = null!;
```

Comment:
I originally used `ShowStoryboard` when I chose the `showReplaceBackgroundStoryboard` variable name.
Because of the conflict, I added config after it, and I will restore it.